cmd/test.go

```diff
@@ -21,7 +21,11 @@ func newTestCmd(rootConfig *RootCommandConfig) *cobra.Command {
 	var testCmd = &cobra.Command{
 		Use:   "test",
 		Short: "Test your project in the local Appsody environment",
-		Long:  `This starts a docker container for your project and runs your test in it.`,
+		Long: `Test your project in the local Appsody environment, starting a container for your project, running the stack provided and user specified tests.
```

Comment:
How about: "Test your project in the local Appsody environment by running tests for the stack and your application."

Reply:
I don't like this.. `Test your project in the local Appsody environment`. But perhaps docs team can help with better words for this.